scripts/extensions/markForUser/src/get-article-actions-bulk.tsx

```diff
@@ -14,7 +15,7 @@ export function getActionsBulkInitialize(superdesk: ISuperdesk) {
 
         const someItemsSpiked = articles.some(({state}) => state === 'spiked');
 
-        if (articles.some(isPersonal) || someItemsLocked || someItemsSpiked) {
+        if (articles.some(isPersonal) || someItemsLocked || someItemsSpiked || !hasPrivilege({mark_for_user: 1})) {
```

Comment:
I'd put `hasPrivilege` check as the first condition in the `if` since it's the most important and would also terminate the `if` sooner when the privilege is not there without having to iterate the items and check their state.